blogango/forms.py

```diff
@@ -50,6 +50,7 @@ class PreferencesForm(forms.ModelForm):
 
     class Meta:
         model = Blog
+        exclude = ()
```

Comment:
I think we need to use fields attribute over there instead of exclude = () like fields = "__all__" as from django 1.8 declaring a model inside meta class also needs to specify an extra attribute fields or exclude